src/main/webapp/app/trial/trial.component.ts

```diff
@@ -250,6 +271,33 @@ export class TrialComponent implements OnInit, AfterViewInit {
     cancelUpdateNote() {
         this.noteEditable = false;
     }
+    updateProtocolNo() {
+        if ( this.trialChosen['protocol_no'].match( /^\d+-\d+$/g ) ) {
```

Comment:
The Regex is a bit different comparing to the one above. Could you create some const to share?

Reply:
Sorry, I should use this one. I forgot to change the one above.